app/forms.py

```diff
@@ -184,8 +210,12 @@ def __init__(self, *args, **kwargs):
 
     def clean_mobile_number(self):
         mobile_number = self.cleaned_data['mobile_number']
-        RegexValidator('^\d{10,11}$', message=_("Enter a valid mobile number in "
-                       "the form 14034228916"))(mobile_number)
+        if mobile_number[0] == '1':
```

Comment:
Consider using `mobile_number.startswith('1')` to avoid running into issues when the mobile number is empty